lib/transaction.ex

```diff
@@ -88,6 +88,23 @@ defmodule Braintree.Transaction do
     end
   end
 
+  @doc """
+  Use a `transaction_id` and optional `amount` to settle the transaction.
+  Use this if `submit_for_settlement` was false while creating the charge using sale.
+
+  ## Example
+      {:ok, transaction} = Transaction.submit_for_settlement("123", %{amount: "100"})
+      transaction.status # 'settling'
+  """
+  @spec submit_for_settlement(String.t, Map.t) :: {:ok, any} | {:error, Error.t}
```

Comment:
The type t for this file is not defined. I am assuming it won't work without it. The rest of method for this module also seem to be using `any`.
If you can point me to a resource where i can get the expected values for all the fields, I can add the type spec and update other methods as well.